content/en/docs/contribute/review/reviewing.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Reviewing pull requests
+content_template: templates/concept
+main_menu: true
+weight: 10
+---
+
+{{% capture overview %}}
+
+Anyone can review a documentation pull request. Visit the [pull requests](https://github.com/kubernetes/website/pulls) section in the Kubernetes website repository to see open pull requests.
+
+Reviewing documentation pull requests is a
+great way to introduce yourself to contributing to the Kubernetes community.
+It helps you learn the code base and build trust with other contributors.
+
+Before reviewing, it's a good idea to:
+
+- Read the  [content guide](/docs/contribute/style/content-guide/) and
+[style guide](/docs/contribute/style/style-guide/) so you can leave informed comments.
```

Comment:
nit: Improved readability over the main page that repeats some of this information.